components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/impl/APIMgtAdminServiceImpl.java

```diff
@@ -50,15 +54,46 @@ public APIMgtAdminServiceImpl(APISubscriptionDAO apiSubscriptionDAO, PolicyDAO p
     }
 
     /**
-     * @see org.wso2.carbon.apimgt.core.api.APIMgtAdminService#getAPISubscriptions(int)
+     * @see APIMgtAdminService#getPoliciesByLevel(PolicyLevel)
+     */
+    @Override
```

Comment:
The purpose of @see annotation is not the same. And by default java doc comments are inherited. That was the reason we agreed to avoid doing this.  We do this as a practice now.